script/run-comp-match

```diff
@@ -232,6 +234,14 @@ def parse_args() -> argparse.Namespace:
         action='store_true',
         dest='no_record',
     )
+    parser.add_argument(
+        '--resolution',
+        help="Set the resolution of the produced video. The format is <width> <height>",
```

Comment:
Specifying the format is redundant -- `argparse` will include that for you:
```
$ ./script/run-comp-match 
usage: run-comp-match [-h] [--duration DURATION] [--no-record]
                      [--resolution width height]
                      archives_dir match_num tla tla
run-comp-match: error: the following arguments are required: archives_dir, match_num, tla

$ ./script/run-comp-match -h
  ...
  --resolution width height
                        Set the resolution of the produced video. The format
                        is <width> <height>
```

```suggestion
        help="Set the resolution of the produced video.",
```

Reply:
Fair point